### DISCUSSION TITLE
Synology - database corrupt after migration to postgresql

### DISCUSSION THREAD

**Discussion by MichalMChudy** [dcm4che/dcm4chee-arc-light#4622] [Q&A] [answered]
Hi,
recently my DSM updated to 7.2.2. After the update, Docker was replaced by Container Manager, along with some other changes. I updated every package according to instructions (MariaDB with data migration, PACS to 5.30.0-1307). However, after the update, neither my archived studies nor sending new ones are available. Fetching studies through PACS-Manager returns error 500.
<img width="966" alt="Bez tytułu2" src="https://github.com/user-attachments/assets/fdb8b06b-7d74-4440-8cc5-7ee42587d949">

After further research it seems like database was not migrated properly. 
<img width="702" alt="Bez tytułu" src="https://github.com/user-attachments/assets/253d10b6-8838-4060-b0fc-24212cea98d6">

Please support, either how to recreate database from saved studies, migrate MariaDB again or reconnect back to MariaDB database  (if this is actual problem)

Below error dumps from docker and dcm4chee
[pacs-db.csv](https://github.com/user-attachments/files/17943709/pacs-db.csv)
[error.txt](https://github.com/user-attachments/files/17943713/error.txt)

Please support.
Michał Chudy

**Comment by MichalMChudy** [comment, marked as answer]
Alright, after long and uneven battle, I figured it out. I recreated database using pgloader and then applied every update on SQL from 5.11 till 5.30. It looks like Synology installer couldn't handle skipping so many versions.

**Comment by vrindanayak** [comment]
We do not manage or maintain Synology packages.
